src/main/scala/it/bitrock/dvs/api/model/AvroConverters.scala

```diff
@@ -69,7 +69,7 @@ object AvroConverters {
     Geography(x.latitude, x.longitude, x.altitude, x.direction)
 
   private def toAirportInfo(x: KAirportInfo): Airport =
-    Airport(x.code, x.name, x.countryName, x.countryIsoCode2, x.timezone, x.latitude, x.longitude, x.gmt)
+    Airport(x.code, x.name, x.countryName, x.countryIsoCode2, x.timezone, x.latitude, x.longitude, x.gmt, x.city)
 
   private def toAirlineInfo(x: KAirlineInfo): Airline =
```

Comment:
Not related to this PR but these methods should be renamed to something like:
```
private def fromAirportInfo(x: KAirportInfo): Airport
private def toAirport(x: KAirportInfo): Airport
```

Reply:
👍

Reply:
👍